trino/sqlalchemy/dialect.py

```diff
@@ -64,6 +64,13 @@ class TrinoDialect(DefaultDialect):
     # Support proper ordering of CTEs in regard to an INSERT statement
     cte_follows_insert = True
 
+    # JSON serializer and deserializer
+    @staticmethod
+    def trino_json_serializer(text):
+        return TrinoJSONType(json.dumps(text))
+    _json_serializer = trino_json_serializer
+    _json_deserializer = None
```

Comment:
Based on the PR description:
> using normal Python json module serialization/deserialization

deserialization should be also done using `json` module, why then here it's `None`?

Reply:
Yes, good eye :)
This is actually because of a minor bug in the SQLAlchemy code. See here: https://github.com/sqlalchemy/sqlalchemy/blob/main/lib/sqlalchemy/sql/sqltypes.py#L2627
If you don't specify a `_json_serializer` or `_json_deserializer` at all in the dialect, then this will throw an error trying to access an attribute that doesn't exist.
However, if you try to set this attribute to just `json.loads`, Python thinks it's an instance method, and expects 2 parameters to be passed (one of which is `self`), and thus causes the downstream call to fail.
I can get around this by setting `_json_deserializer = staticmethod(json.loads)` if you think this is more clear. I can also just add a comment to this effect.